src/Web/Sfa.Das.Sas.Web/Controllers/ApprenticeshipController.cs

```diff
@@ -56,20 +56,38 @@ public ActionResult SearchResults(ApprenticeshipSearchQuery query)
         // GET: Standard
         public ActionResult Standard(string id, string keywords)
         {
-            var response = _mediator.Send(new GetStandardQuery { Id = id, Keywords = keywords });
+            var response = _mediator.Send(new GetStandardQuery {Id = id, Keywords = keywords});
 
-            if (response.StatusCode == GetStandardResponse.ResponseCodes.InvalidStandardId)
+            switch (response.StatusCode)
             {
-                _logger.Info("404 - Attempt to get standard with an ID below zero");
-                return HttpNotFound("Cannot find any standards with an ID below zero");
-            }
+                case GetStandardResponse.ResponseCodes.InvalidStandardId:
+                {
+                    _logger.Info("404 - Attempt to get standard with an ID below zero");
+                    return HttpNotFound("Cannot find any standards with an ID below zero");
+                }
+
+                case GetStandardResponse.ResponseCodes.StandardNotFound:
+                {
+                    var message = $"Cannot find standard: {id}";
+                    _logger.Warn($"404 - {message}");
 
-            if (response.StatusCode == GetStandardResponse.ResponseCodes.StandardNotFound)
-            {
-                var message = $"Cannot find standard: {id}";
-                _logger.Warn($"404 - {message}");
+                    return new HttpNotFoundResult(message);
+                }
 
-                return new HttpNotFoundResult(message);
+                case GetStandardResponse.ResponseCodes.AssessmentOrgsEntityNotFound:
+                {
+                    var message = $"Cannot find assessment organisations for standard: {id}";
+                    _logger.Warn($"404 - {message}");
+                    return new HttpNotFoundResult(message);
```

Comment:
I think we should just return the response instead of raising a HttpNotFoundresult error. (because we don't want page not found when there are no assessment org found for a standard).

Reply:
Yes, that sounds right.  Will do